ProcessesApi/V1/Helpers/ChangeOfNameHelper.cs

```diff
@@ -0,0 +1,16 @@
+using ProcessesApi.V1.Services.Exceptions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ProcessesApi.V1.Helpers
+{
+    public static class ChangeOfNameHelper
+    {
+        public static void ValidateOptionalFormData(Dictionary<string, object> requestFormData, List<string> expectedFormDataKeys)
+        {
+            if (!requestFormData.ContainsKey("firstName") && !requestFormData.ContainsKey("surname") &&
```

Comment:
You can just do
```
var isValid = requestFormData.Any(x => expectedFormDataKeys.Contains(x));
```

and then it can be reused so can go back to the normal helper file